src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,333 @@
+    private static final String MESSAGE_DIVIDER = "===============================================================================";
+    private static final String MESSAGE_DIVIDER_LIST = "=====================================LIST======================================";
+    private static final String MESSAGE_UNKNOWN_COMMAND = "Unknown Command";
+    private static final String ERROR_MESSAGE_BYE = "";
+    private static final String ERROR_MESSAGE_LIST = "";
+    private static final String ERROR_MESSAGE_MARK = "Syntax for (un)mark \n\t>>> (un)mark <item index number> \nNote: item index must exist in the current list";
+    private static final String ERROR_MESSAGE_TODO = "Syntax for todo \n\t>>>todo <task>";
+    private static final String ERROR_MESSAGE_DEADLINE = "Syntax for deadline \n\t>>>deadline <task> / <date of deadline>";
+    private static final String ERROR_MESSAGE_EVENT = "Syntax for event \n\t>>>event <task> / <date of event>";
+    private static final String ERROR_MESSAGE_DELETE = "Syntax for delete \n\t>>> delete <item index number> \nNote: item index must exist in the current list";
+
+    private static final String STORAGE_DIRECTORY = "./storage";
+    private static final String STORAGE_FILE = "./storage/savedTaskList.txt";
+    private static final String MESSAGE_NEW_FILE = "Storage file created";
+    private static final String MESSAGE_OVERWRITE_FILE = "Storage file overwritten";
+    private static final String MESSAGE_LOAD_FILE = "Storage file loaded";
+    private static final String MESSAGE_NO_FILE = "There is no existing file!";
+    private static final String MESSAGE_WRONG_FILE = "The storage files entry is invalid. Please save to overwrite";
+    /**
+     * Global Variables Section
+     */
+    static Scanner in = new Scanner(System.in);
+
+    static ArrayList<Task> tasksList = new ArrayList<>(MAXIMUM_NUMBER_OF_TASKS);
+    static boolean isRunning = true;
+
+    /**
+     * Main Function
+     *
+     * @param args
+     */
+    public static void main(String[] args) {
+        helloMessage();
+        load();
+        while (isRunning) {
+            System.out.print(MESSAGE_PROMPT);
+            parseCommand();
+        }
+    }
+
+    /**
+     * Global Methods Sections
+     */
+
+    private static void showToUser(String... message) {
+        for (String m : message) {
+            System.out.println(m);
+        }
+    }
+
+    private static void showToUserDivider(String... message) {
+        showToUser(MESSAGE_DIVIDER);
+        showToUser(message);
+        showToUser(MESSAGE_DIVIDER);
+    }
+
+    private static void parseCommand() {
+        String line = in.nextLine();
+        String[] parsedLine = line.split(" ");
+        String commandWord = parsedLine[0].toLowerCase();
+
+        switch (commandWord) {
+        case "bye":
+            exit();
+            break;
+        case "list":
+            list();
+            break;
+        case "mark":
```

Comment:
Consider adding explicit fallthrough comments here.
```suggestion
        case "mark": // Fallthrough
```

https://se-education.org/guides/conventions/java/basic.html#layout